modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/robotic.dm

```diff
@@ -7,10 +7,8 @@
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | ERT_SPAWN | RACE_SWAP | SLIME_EXTRACT
 	reagent_flags = PROCESS_SYNTHETIC
 	coldmod = 0.5
-	burnmod = 1.1
-	heatmod = 1.2
-	brutemod = 1.1
-	siemens_coeff = 1.2 //Not more because some shocks will outright crit you, which is very unfun
+	heatmod = 3
+	siemens_coeff = 1.4 //Not more because some shocks will outright crit you, which is very unfun
```

Comment:
1.3 would be better. 1.4 can crit I believe.
Heatmod should also probably be 2 instead of 3, since heat damage is much more expensive to repair overall; and stays as a lingering effect for much longer
probably should reduce their coldmod to 0.2 in exchange

Reply:
1.4 is no longer crit because of the health increase , heatmod is rarely a damage type that is used and for the advantages they have they shouldn't it so low. Coldmod doesn't need to be adjusted at all really, going lower would make them even more space proof which is not what i intend.